etl/steps/data/garden/unicef/2024-07-30/child_migration.meta.yml

```diff
@@ -0,0 +1,115 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Migration
+        - Violence Against Children & Children's Rights
+    unit: people
+    short_unit: ""
+    processing_level: minor
+  unicef_descriptions:
+    refugee: |-
+      Person who are outside their country of nationality or habitual residence, who cannot return due to a well-founded fear of being persecuted because of their race, religion, nationality, membership of a particular social group or political opinion. This number only accounts those who have been recognized as refugees or find themselves in refugee-like situations.
+      This data is sourced from: *United Nations High Commissioner for Refugees, Global Trends: Forced Displacement in 2018, UNHCR, Geneva, 2019.* Share of under 18 from UNHCR unpublished data, cited with permission.
```

Comment:
Here I also see the 'unpublished' bit, but I don't see you use a file coming from another place (or at least it's not documented)

Reply:
Oh, so the data is imported from UNICEF, who have consolidated it from a number of different sources (one of which is the unpublished UNHCR data). I felt like the "description from producer" field was a good place to put this information, but I can take it out if it is confusing or should be somewhere else. 

The data is published by UNICEF, so there is no unpublished data here :)